packages/styles/scss/components/metrics-card.scss

```
@@ -0,0 +1,60 @@
+@use 'sass:map';
+@use '../config.scss' as c;
+@use '../utilities/custom-properties.scss' as cv;
+
+$card-colors: (
+  'red': cv.get-var('metrics-red'),
+  'hoverRed': cv.get-var('hover-red'),
+  'selectColor': cv.get-var('layer-02'),
```

This can just refer to `layer` which in turn corresponds to the current layer being used. This would enable it to have the right color for all different layers.

```suggestion
  'selectColor': cv.get-var('layer'),
```